src/client_registration.rs

```diff
@@ -1,34 +1,24 @@
 use serde::{Deserialize, Serialize};
 use serde_json::{json, Value};
 
-use crate::lib::{Debug, String, ToString};
+use crate::lib::{Debug, String};
 
 #[derive(Debug, Clone, Deserialize, Serialize)]
-struct Workload {
```

Comment:
I included this temporarily because it was not exposed by `reference-kbs` IIRC.

Since we are now writing `keybroker`, can we include this structure from it?

BTW maybe we can remove `new()` and implement only `register` that returns the `keybroker` structure.